src/app/modules/recipe-list/containers/recipe-list.component.ts

```diff
@@ -18,7 +24,7 @@ export class RecipeListComponent implements OnInit {
   defaultPerPageOptions = defaultPerPageOptions;
   defaultSortBy = defaultSortBy;
 
-  public recipes: Recipe[] = [];
+  public recipes: Observable<Recipe[]> = this.store.select(selectRecipes);
```

Comment:
Move ` = this.store.select(selectRecipes)` to `ngOnInit`
```suggestion
  public recipes$: Observable<Recipe[]>;
```